imap/user.c

```diff
@@ -237,50 +249,97 @@ static int user_deletesieve(const char *user)
     return 0;
 }
 
-EXPORTED int user_deletedata(const char *userid, int wipe_user)
+static const char *wipe_user_file_suffixes[] = {
+    FNAME_SEENSUFFIX,
+    FNAME_MBOXKEYSUFFIX,  /* XXX  what do we do about multiple backends? */
+    NULL
+};
+
+static const char *user_file_suffixes[] = {
+    FNAME_DAVSUFFIX,      /* even if DAV is turned off, this is fine */
+    FNAME_SUBSSUFFIX,
+
+    /* XXX: one could make an argument for keeping the counters
+     * file forever, so that UIDVALIDITY never gets reused. */
```

Comment:
Not really - unless you stored it by username rather than inbox-uniqueid, because otherwise the newly created user isn't going to find it anyway.

Reply:
This comment was copied from the original user_deletedata() function.  I'll remove it since it no longer applies (once the user has been relocated)